### ISSUE TITLE
Cannot use it on Catalina - "Please allow Accessibility"

### ISSUE THREAD

**Issue by RalphPastel972** [ian4hu/Clipy#41]
Hello,

I downloaded this updated version of Clipy because the original one stopped working for some reasons (nothing got pasted anymore when using it)

With this version, each time I try to paste, I get the "Please allow Accessibility." message, even if Clipy is actually ticked as an allowed Accessibility App in the the Security & Privacy feature of System Preferences.
Actually, I cannot untick it (which is weird). And it is ticked by default when I install this version

I have no idea what to do.

I use mac OS Catalina Version 10.15.1 and Clipy V1.2.9 beta 12

Any advice for me?
Thank you very much
Regards

**Comment by RalphPastel972** [comment]
I found the solution
Please, have a look here: https://apple.stackexchange.com/questions/372968/i-cant-check-or-uncheck-items-in-security-privacy-accessibility-on-macos-ca

**Comment by RalphPastel972** [comment]
I found the solution
Please, have a look here: https://apple.stackexchange.com/questions/372968/i-cant-check-or-uncheck-items-in-security-privacy-accessibility-on-macos-ca
